pkg/metaserver/agent/metric/helper/device.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2022 The Katalyst Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package helper
+
+import (
+	"github.com/kubewharf/katalyst-core/pkg/metaserver/agent/metric/types"
+	"github.com/kubewharf/katalyst-core/pkg/metrics"
+	"github.com/kubewharf/katalyst-core/pkg/util/general"
+	metricutil "github.com/kubewharf/katalyst-core/pkg/util/metric"
+)
+
+func GetDeviceMetricWithTime(metricsFetcher types.MetricsFetcher, emitter metrics.MetricEmitter, metricName string, devName string) (metricutil.MetricData, error) {
```

Comment:
we need make an appointment here (it may be a little confusing now): for a certain metric, when should we put the get-logic in `helper` package? in the previous design,  `helper` package will only be used when we want to assemble or re-calculate based on the raw-metric, but it seems not for now; and we'd better add comments in helper.consts.go @nightmeng